workflows/yelp.com/workflows.arazzo.json

```diff
@@ -31,7 +31,7 @@
       "steps": [
         {
           "stepId": "search-business-by-phone",
-          "description": "Search for a business using a phone number.",
+          "description": "Search for a business using its phone number.",
```

Comment:
We need to look for a first business

"businesses": "$response.body#/businesses/0"